invenio_curations/services/generators.py

```diff
@@ -60,6 +62,59 @@ def _condition(self, request=None, **kwargs):
         return False
 
 
+class EntityReferenceServicePermission(Generator):
```

Comment:
so if i understand correctly, this basically "forwards" the permission check to the service responsible for the referenced entity?

Reply:
Yeah. An entity will be resolved and passed to the generators of the specified permission.